app.py

```diff
@@ -11,6 +11,8 @@
 def index():
     item_list = []
     list_id = '5f56323626c33d81cd98b386'
+    url = f'https://api.trello.com/1/lists/{list_id}/cards?key={os.getenv("KEY")}&token={os.getenv("TOKEN")}'
+    print("Trying url: {}", url)
```

Comment:
Looks like this is just for testing so we can lose it, but might be nice to keep the url variable and use it in our request just to make that line a little more readable

Reply:
will do, ty